lib/std/c/freebsd.zig

```diff
@@ -629,6 +630,191 @@ comptime {
     std.debug.assert(@sizeOf(kinfo_vmentry) == KINFO_VMENTRY_SIZE);
 }
 
+pub const WMESGLEN = 8;
+pub const LOCKNAMELEN = 8;
+pub const TDNAMLEN = 16;
+pub const COMMLEN = 19;
+pub const MAXCOMLEN = 19;
+pub const KI_EMULNAMELEN = 16;
+pub const KI_NGROUPS = 16;
+pub const LOGNAMELEN = 17;
+pub const LOGINCLASSLEN = 17;
+
+pub const KI_NSPARE_INT = 2;
+pub const KI_NSPARE_LONG = 12;
+pub const KI_NSPARE_PTR = 5;
+
+pub const RUSAGE_SELF = 0;
+pub const RUSAGE_CHILDREN = -1;
+pub const RUSAGE_THREAD = 1;
+
+pub const proc = opaque {};
+pub const thread = opaque {};
+pub const vnode = opaque {};
+pub const filedesc = opaque {};
+pub const pwddesc = opaque {};
+pub const vmspace = opaque {};
+pub const pcb = opaque {};
+pub const lwpid_t = i32;
+pub const fixpt_t = u32;
+pub const vm_size_t = if (builtin.cpu.arch.ptrBitWidth() == 32) u32 else u64;
+pub const segsz_t = if (builtin.cpu.arch.ptrBitWidth() == 32) i32 else i64;
+
+pub const itimerval = extern struct {
+    it_interval: timeval,
+    it_value: timeval,
+};
+
+pub const pstats = extern struct {
+    p_cru: rusage,
+    p_timer: [3]itimerval,
+    p_prof: extern struct {
+        pr_base: u8,
+        pr_size: c_ulong,
+        pr_off: c_ulong,
+        pr_scale: c_ulong,
+    },
+    p_start: timeval,
+};
+
+pub const user = extern struct {
+    u_stats: pstats,
+    u_kproc: kinfo_proc,
```

Comment:
```suggestion
    stats: pstats,
    kproc: kinfo_proc,
```

zig std lib convention is to erase prefixes like this on all fields.